osu.Game/Configuration/OsuConfigManager.cs

```diff
@@ -20,6 +20,8 @@ protected override void InitialiseDefaults()
             Set(OsuSetting.DisplayStarsMinimum, 0.0, 0, 10);
             Set(OsuSetting.DisplayStarsMaximum, 10.0, 0, 10);
 
+            Set(OsuSetting.SelectionRandomType, SelectionRandomType.Random);

```

Comment:
I think it's totally fine to just make your permutation-based randomness the default (and only) random selection method. @peppy ?